src/hott/03-equivalences.rzk.md

```diff
@@ -813,6 +813,28 @@ identifications. This defines `#!rzk eq-htpy` to be the retraction to
   ( f g : (x : X) → A x)
   : ((x : X) → f x = g x) → (f = g)
   := first (first (funext X A f g))
+
+#def left-cancel-is-equiv uses (funext)
```

Comment:
@StiephenPradal can you explain what you defined these for? They seem to be essentially `eq-htpy` applied to `has-section-is-equiv` and `has-retraction-is-equiv` defined earlier in the file.

Reply:
I have defined these paths to make some proofs look more compact. If they are not useful for the library, we can remove them and I will use `eq-hpty` where they appear instead.